app/geant-exporter/GeantPhysicsTableWriter.cc

```diff
@@ -350,9 +350,9 @@ void GeantPhysicsTableWriter::fill_energy_loss_tables(
                         ImportTableType::dedx_subsec);
         this->add_table(process.DEDXunRestrictedTable(),
                         ImportTableType::dedx_unrestricted);
-        this->add_table(process.IonisationTable(), ImportTableType::ionisation);
+        this->add_table(process.IonisationTable(), ImportTableType::ionization);
         this->add_table(process.IonisationTableForSubsec(),
-                        ImportTableType::ionisation_subsec);
+                        ImportTableType::ionization_subsec);
```

Comment:
Ah this is the downside of changing spellings... I'm torn between trying to keep the Geant4 functions consistent with the `ImportX` names and keeping the code that we define consistent.

Reply:
Yeah. I thought the same when renaming that part. My reasoning is that since we did not keep the process and model names exactly as in Geant4 anyway in our enums, we might as well just go ahead and make it fully consistent.

Reply:
Good call. Thanks!